cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -186,8 +197,13 @@ func (s *initramfsMountsSuite) SetUpTest(c *C) {
 	restore = func() { dirs.SetRootDir("") }
 	s.AddCleanup(restore)
 
+	// use a specific time for all the assertions, in the future so that we can
+	// set the timestamp of the model assertion to something newer than now, but
+	// still older than the snap declarations by default
+	s.snapDeclAssertsTime = time.Now().Add(60 * time.Minute)
```

Comment:
maybe it would be enough to Truncate here, but maybe is less clear given the distance?

Reply:
hmm yeah I think it would be less clear, it would save some code but that time would be quickly lost in thinking there is a bug around not truncating snapDeclAssertsTime in the places we compare it to, I'm inclined to not do this